components/ongage/actions/find-subscriber/find-subscriber.js

```diff
@@ -0,0 +1,29 @@
+const ongage = require('../../ongage.app.js');
```

Comment:
Huge thanks for this PR @compwright !

We just published a new `.eslintrc` file to enforce consistent ESLint rules across components. Could you do me a favor and do the following?

1. Pull latest `master` / rebase your fork with the latest changes
2. Run `npm ci` from the repo root to install ESLint and related dependencies?
3. Either use an ESLint plugin in your editor to automatically format the files based on the rules, or run `eslint --fix <file>` on each file in this commit.